app/src/main/java/org/simple/clinic/reassignPatient/ReassignPatientSheet.kt

```diff
@@ -137,5 +142,11 @@ class ReassignPatientSheet : BaseBottomSheet<
     @Parcelize
     data object SelectFacility : ScreenRequest()
   }
+
+  @Parcelize
+  data class SelectedNavigation(
```

Comment:
Not sure about this name. It's generic and doesn't indicate the action we are thinking. Let's think of a new name for it?